### DISCUSSION TITLE
.reality in model viewer

### DISCUSSION THREAD

**Discussion by Dovatech** [google/model-viewer#2637] [Q&A]
### Description

Hello, can I use .reality file in model viewer?

### Browser Affected

- [x] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari

### OS

- [ ] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

- [ ] WebXR
- [ ] SceneViewer
- [x] QuickLook

**Comment by elalish** [comment]
Yes, you can put a .reality file as the `ios-src`.

**Comment by ti-hardikshah** [comment]
I had face one issue, when I tried in Reality composer it looks good with all animation and behavior set. but in model viewr all meshup not working animation !! is there any solution? 
Size can be effect ? .reality size is 40 mb
